test/identifier/single-statement.spec.ts

```diff
@@ -65,6 +65,32 @@ describe('identifier', function () {
       expect(actual).to.eql(expected);
     });
 
+    ['DATABASE', 'SCHEMA'].forEach((type) => {
+      describe(`identify "CREATE ${type}" statements`, () => {
+        const sql = `CREATE ${type} Profile;`;
+        it('should identify statement', function () {
+          const actual = identify(sql);
+          const expected = [
+            {
+              start: 0,
+              end: sql.length - 1,
+              text: sql,
+              type: `CREATE_${type}`,
+              executionType: 'MODIFICATION',
+            },
+          ];
+
+          expect(actual).to.eql(expected);
+        });
+
+        it('should throw error for sqlite', () => {
+          expect(() => identify(sql, { dialect: 'sqlite' })).to.throw(
+            `Expected any of these tokens (type="keyword" value="TABLE") or (type="keyword" value="VIEW") or (type="keyword" value="TRIGGER") or (type="keyword" value="FUNCTION") or (type="keyword" value="INDEX") instead of type="keyword" value="${type}" (currentStep=1)`
+          );
+        });
+      });
+    });
```

Comment:
I accidentally removed the standalone `CREATE DATABASE` test when grouping trigger / function tests in 324bfb9d1b17f13a7f75fe60c93961d628714379, which is why you don't see the removal / addition similar to `DROP DATABASE`, and just the addition.